.github/workflows/dev-workflow.yml

```
@@ -222,60 +222,60 @@ jobs:
-  #         MIGRATE: false
-  #       run: |
-  #         make docker.build
-  #         make docker.login
-  #         make docker.push
+  fetcher:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: packages/fetcher
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)